CHANGES.md

```diff
@@ -0,0 +1,10 @@
+1.1.0 / ??-??-2017
+==================
+
+* Add PDF output support
+* Add support to customize the output format with a `format` request parameter
+
+1.0.0 / ??-??-2017
```

Comment:
You should update this when you're about to push a release, as I couldn't know when you'll do that (also, couldn't find a release date for 1.0). But hopefully today? ;) Also, `package.json` must be updates at the same time.